src/components/Body.js

```diff
@@ -1,9 +1,35 @@
 /* eslint-disable class-methods-use-this */
 
 import React from "react";
+import PropTypes from "prop-types";
 
 export default class Body extends React.Component {
   render() {
-    return <h2>Words</h2>;
+    const markSchemeArray = this.props.markSchemeElements.map((element, i) => (
+      <li key={i}> {element.text} </li>
+    ));
+    return (
+      <div>
+        <button data-testid="swipeButton" onClick={this.props.swipe}>
```

Comment:
Please add a comment here making clear that this button will need to be removed once we have implemented a listener for a swipe event.

Reply:
done!